src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDone() {
+        isDone = true;
+    }
+
+    public void printTask(){
+         System.out.println(description);
+    }
+
+    public String getStatusIcon() {
+        // mark done task with X if done
```

Comment:
I think this method only returns the String. It doesn't mark the done task with X. I think the method that's calling this method is the one that marks the task.